src/day-1-toy/modules.py

```diff
@@ -6,26 +6,27 @@
 # See docs for the sys module: https://docs.python.org/3.7/library/sys.html
 
 # Print out the command line arguments in sys.argv, one per line:
-
+print(sys.argv)
```

Comment:
This will output a list of string arguments that were passed in the program's execution. How would you output each string argument individually on each line?